projects/packages/my-jetpack/_inc/data/recommendations-section/use-recommendations-section.ts

```diff
@@ -0,0 +1,20 @@
+import { __ } from '@wordpress/i18n';
+import { QUERY_SAVE_EVALUATION_KEY, REST_API_SAVE_EVALUATION_RECOMMENDATIONS } from '../constants';
+import useSimpleMutation from '../use-simple-mutation';
+
+const useRecommendationsSection = () => {
+	const { mutate: saveEvaluation } = useSimpleMutation( {
```

Comment:
What would you think about moving the `startEvaluation` mutation to this file as well? That way this hook could return both of them?

Either that or this hook could be moved to the WelcomeFlow component as it doesn't seem to have any more logic than the `startEvaluation` mutation that is defined there

Reply:
I plan to extend this hook with implementation of recommendations section (to show/hide/reset the section). Good point that this mutation can be moved to the hook altogether!